### PR TITLE
Migrate code to ES6 syntax (apps/private-blogging)

### PR DESCRIPTION
**Pull request by tiendq**
issue #9589
- update var to const and let

- [x] There's a clear use-case for this code change
- [x] Commit message has a short title & references relevant issues
- [x] The build will pass (run `npm test`)

### CODE REVIEW

File: core/server/apps/private-blogging/index.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
     router = require('./lib/router'),
     registerHelpers = require('./lib/helpers'),
     // routeKeywords.private: 'private'
-    PRIVATE_KEYWORD = 'private',
-    checkSubdir;
+    PRIVATE_KEYWORD = 'private';
 
-checkSubdir = function checkSubdir() {
-    var paths;
+let checkSubdir = function checkSubdir() {
```
Comment (vikaspotluri123) [inline core/server/apps/private-blogging/index.js]: Do we need to redundantly define the function here? More of a question for @kirrg001, but just thinking out loud :smile

Comment (tiendq) [reply core/server/apps/private-blogging/index.js]: You could ask this question earlier, I don't invent it in this PR :)

Comment (kirrg001) [reply core/server/apps/private-blogging/index.js]: Definitely redundant, good catch 👍 

@tiendq No need to remove the function, the PR is only about ES6 :)

**Review by kirrg001 — APPROVED** [review]
Thanks!

**Pull request merged**